docs/source/settings.rst

```diff
@@ -210,6 +210,16 @@ you're sure of the repercussions for sync workers. For the non sync
 workers it just means that the worker process is still communicating and
 is not tied to the length of time required to handle a single request.
 
+.. _startup-timeout:
+
+startup_timeout
+~~~~~~~~~~~~~~~
+
+* ``--startup-timeout INT``
+* ``30``
+
+Timeout for workers startup.
```

Comment:
It would be nice if you specify in the description what time units the value has (milliseconds/seconds/minutes?).
Reasonable question from people who will use it: is it timeout per worker or aggregation of all workers startup time? Need to specify it clearly in the docs.

Also you better to point out in the docs explicitly how `--startup-timeout` correlates with `--timeout` if they both are specified.